src/PlanarHarmonicBarrierGPU.cuh

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2018-2020, Michael P. Howard
+// Copyright (c) 2021-2024, Auburn University
+// Part of azplugins, released under the BSD 3-Clause License.
+
+/*!
+ * \file PlanarHarmonicBarrierGPU.cuh
+ * \brief Declaration of kernel drivers for PlanarHarmonicBarrierGPU
+ */
+
+#ifndef AZPLUGINS_PLANAR_HARMONIC_BARRIER_GPU_CUH_
+#define AZPLUGINS_PLANAR_HARMONIC_BARRIER_GPU_CUH_
+
+#include "hoomd/HOOMDMath.h"
+#include <cuda_runtime.h>
+
+namespace hoomd
+    {
+namespace azplugins
+    {
+namespace gpu
+    {
+
+//! Kernel driver to evaluate PlanarHarmonicBarrierGPU force
+cudaError_t compute_harmonic_force(Scalar4* d_force,
```

Comment:
This function and the corresponding kernel should be renamed something more descriptive & specific to the class:
```suggestion
cudaError_t compute_force_planar_harmonic_barrier(Scalar4* d_force,
```